File: rust/fizzbuzz.rs
fn fizzbuzz(n: usize) -> String
{
    match n {
        _ if (n % 15) == 0 => "FizzBuzz".to_string(),
        _ if (n % 3)  == 0 => "Fizz".to_string(),
        _ if (n % 5)  == 0 => "Buzz".to_string(),
        _                  => n.to_string(),
    }
}


fn main()
{
    for n in 1..101 {
        print!("{} ", fizzbuzz(n));
    }
    println!("");
}

File: python3/fizzbuzz.py
def fizzbuzz(n):
    f = lambda i : \
            "FizzBuzz" if (i % 15) == 0 \
            else "Fizz" if (i % 3) == 0 \
            else "Buzz" if (i % 5) == 0 \
            else i

    return list(map(f, range(1, n + 1)))


def main():
    n = int(input("Input a number: "))
    ns = fizzbuzz(n)
    print(ns)


if __name__ == '__main__':
    main()
